Add CN global_conf

htdvisser opened the pull request:
Resolves #20

htdvisser committed:
Add CN global_conf

htdvisser commented on CN-global_conf.json:
```diff
@@ -0,0 +1,204 @@
+			"enable": true,
+			"type": "SX1255",
+			"freq": 486600000,
+			"rssi_offset": -176.0,
```
@tftelkamp: this is `-166.0` in the other configs, is this change ok?

htdvisser commented on CN-global_conf.json:
```diff
@@ -0,0 +1,204 @@
+			"enable": true,
+			"type": "SX1255",
+			"freq": 487400000,
+			"rssi_offset": -176.0,
```
@tftelkamp: this is `-166.0` in the other configs, is this change ok?

tftelkamp replied:
This number should be provided by the gateway manufacturer. I don't know what a good default is for SX1255 based gateways.

jpmeijers reviewed (commented):
I don't see anything obviously wrong with this config. The most important part is that we have the frequencies and TX powers correct. What the other parameters are concerned, let's leave them as is until someone comments on it.

The frequencies seem correct to me.
I can't comment on the TX powers yet. Will do a bit of investigation on it.

RimeLink commented:
We think the TxFreq shoud modified as:
"tx_freq_min": 500300000
"tx_freq_max":509700000

otherwise the gateway maybe REJECT emit packet of the RX2 that on 505.3MHz

jpmeijers commented:
I agree that we should change the TX frequency limits to @RimeLink's suggestion. It is up to the network to decide which downlink frequency should be used. These limits should only prevent transmissions on out of band frequencies.
```
"tx_freq_min": 500300000
"tx_freq_max": 509700000
```

jpmeijers committed:
Update CN-global_conf.json

Change TX frequency limits

jpmeijers commented:
Looking at this again:
"The LoRaWAN can be used in the Chinese 470-510MHz band..." - LoRaWAN Regional Parameters
From this we should perhaps use:
```
"tx_freq_min": 470000000
"tx_freq_max": 510000000
```
Then it will be inline with what the other configs do - allowing transmissions anywhere in the allowed band.

jpmeijers committed:
Update CN-global_conf.json

Change TX frequency limits to:
"tx_freq_min": 470000000
"tx_freq_max": 510000000

Remember we should allow gateways to be used as beacons
